app/serializers/user_serializer.rb

```diff
@@ -43,10 +23,6 @@ def max_subjects
     @model.subject_limit
   end
 
-  def uploaded_subjects_count
```

Comment:
I thought we'd probably want that in there still, maybe as an optional attribute?